packages/@aws-cdk/aws-eks/lib/aws-auth-mapping.ts

```diff
@@ -1,4 +1,6 @@
-
+/**
+ * AwsAuth mapping.
+ */
 export interface Mapping {
```

Comment:
Don't know that `Mapping` is a good name for it, a little too generic for the purpose it seems to serve.

Reply:
That's a good point. Let me deprecate it.

Reply:
Done